### PR TITLE
Initial dev

### PR DESCRIPTION
**Pull request by jyothishp994**
This is the initial dev of Nox component library.
Please check the hosted app in below url.
https://nox-ui.netlify.app/

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,717 @@
...
+    font-family: 'Roboto', sans-serif;
+    font-family: 'Rowdies', cursive;
+
+}
+
+.headermain{
+    height: 4rem;
+    display: flex;
+    flex-direction: row;
+    box-shadow: 0 8px 6px -6px lightblue;background-color: lightblue;
```
Comment (pnchinmay) [inline css/style.css:16]:
```suggestion
    box-shadow: 0 8px 6px -6px lightblue;
    background-color: lightblue;
```

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,717 @@
...
+    object-fit: contain;
+}
+.footer-container{
+
+    height:auto;
+background-color: black;
+margin-bottom: 0;
+color: white;
+font-family: 'Heebo', sans-serif;
+font-family: 'PT Sans', sans-serif;
```
Comment (pnchinmay) [inline css/style.css:49]: Take care of indentation
Google style guide suggests indentation of 2 spaces for HTML and css

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,717 @@
...
+text-align: center;
+
+    
+}
+.main-container-body-desc,.main-container-body-item-desc{
+    font-family: 'Heebo', sans-serif;
+font-family: 'PT Sans', sans-serif;
+font-size: 1rem;
+}
```
Comment (pnchinmay) [inline css/style.css:84]: same here

**Review by pnchinmay — CHANGES_REQUESTED** [review]

**Review by MonikaJhingan — APPROVED** [review]
The code looks good to be merged.

File: css/style.css (hunk 3 above)
Comment (jyothishp994) [reply css/style.css:84]: Taken care, Thanks for comments

File: css/style.css (hunk 1 above)
Comment (jyothishp994) [reply css/style.css:16]: Code is now indented, thanks for the comments

File: css/style.css (hunk 2 above)
Comment (jyothishp994) [reply css/style.css:49]: Taken care, thanks for comments

**Pull request merged**